### PR TITLE
Bug/673 alignment

### PR DESCRIPTION
**Pull request by aaronsleeper**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
+}
```
Comment (esotrope) [inline package-lock.json]: Not sure why this file is here. I think the only one we need is webapp/src/main/webapp/package-lock.json

File: webapp/src/main/webapp/package-lock.json [hunk 2]
```diff
@@ -884,6 +893,11 @@
         "hoek": "2.16.3"
       }
     },
+    "bootstrap": {
+      "version": "3.3.7",
+      "resolved": "https://registry.npmjs.org/bootstrap/-/bootstrap-3.3.7.tgz",
+      "integrity": "sha1-WjiTlFSfIzMIdaOxUGVldPip63E="
+    },
```
Comment (esotrope) [inline webapp/src/main/webapp/package-lock.json]: Curious why we need this. Was this put in manually or just happened when npm build was run?

File: package-lock.json (hunk 1 above)
Comment (bgsavage) [inline package-lock.json]: Agreed, please remove this file.

File: webapp/src/main/webapp/src/primeng-overrides.less [hunk 3]
```diff
@@ -164,6 +166,18 @@ body .ui-widget.ui-datatable {
...
+    &.level-up {
+      border-right: 1px solid @white;
+    }
+    &.level-down {
+      display: flex !important;
+      border-bottom: 1px solid @table-border-color;
+      border-left: none;
+      overflow: hidden;
+    }
+  }
```
Comment (aaronsleeper) [inline webapp/src/main/webapp/src/primeng-overrides.less]: admin should have the same styles as front

File: webapp/src/main/webapp/package-lock.json (hunk 2 above)
Comment (aaronsleeper) [inline webapp/src/main/webapp/package-lock.json]: i didn't make that change. how do i un-include it

Comment (bgsavage) [inline webapp/src/main/webapp/package-lock.json]: Yes, if this is the version of bootstrap @aaronsleeper  wants to lock in.

Comment (aaronsleeper) [inline webapp/src/main/webapp/package-lock.json]: i haven't seen it break anything and i didn't include it . i'm fine keeping it up to date

**Pull request merged**